### DISCUSSION TITLE
Arrays

### DISCUSSION THREAD

**Discussion by richallyson** [balta-io/2801#66] [Perguntas e Respostas] [answered]
Salve Balta! Eu aqui de novo tirando mais uma dúvida. Mas dessa vez é sobre isso aqui abaixo:
![image](https://user-images.githubusercontent.com/24733986/164120460-76cc0b5d-9eda-4644-a637-f6ccff83adf6.png)

Ai dentro do For eu estou instanciando o valor do id de funcionário do jeito que você ensinou na aula. Porém, acima disso, eu usei outra forma, que é a forma como sempre utilizo (está comentada). Existe alguma diferença de uma pra outra, ou as duas no fim são a mesma coisa só que feita de formas diferente?

Brigadão!

**Comment by andrebaltieri** [comment, marked as answer]
Neste caso, não, pois os itens já existem no array, agora se você tentar atribuir um valor ao `Id` de um item que não existe, vai dar erro.

**Comment by richallyson** [comment]
Obrigado!
